### PR TITLE
fix api auth me response did not wrap with data

### PR DESCRIPTION
**Pull request by koshuang**
#73

### CODE REVIEW

**Comment by coomysky** [comment]
r+ thanks!

File: Modules/Auth/Http/Controllers/AuthController.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function store(Request $request)
      */
     public function me()
     {
-        return response()->json(new UserResource(auth()->user()));
+        return new UserResource(auth()->user());
```
Comment (ericHao22) [inline Modules/Auth/Http/Controllers/AuthController.php:68]: `AuthController`的測試應該也要改

**Review by ericHao22 — APPROVED** [review]
r+ with a comment, thanks!!

File: Modules/Auth/Http/Controllers/AuthController.php (hunk 1 above)
Comment (koshuang) [reply Modules/Auth/Http/Controllers/AuthController.php:68]: fix 了，謝謝！

**Pull request merged**